ruff.schema.json

```diff
@@ -1554,6 +1554,7 @@
         "PIE8",
         "PIE80",
         "PIE807",
+        "PL",
         "PLC",
```

Comment:
To be clear, does `PLC` still work to select the `PLC` codes? Or just `PL`?

Reply:
Yes PLC, PLE, PLR, PLW all still work as previously ... it's just that PL now also works ... i.e. the change is backwards compatible.